automation-tutorial/pom.xml

```diff
@@ -72,6 +72,12 @@
         <plugin>
           <artifactId>maven-site-plugin</artifactId>
           <version>3.7.1</version>
+        </plugin> 	
+
+
+<plugin>
```

Comment:
I noticed this is a higher version for the maven-compiler-plugin.
There is version 3.7.1. Will be a good idea to check maven central for the latest version.

Kindly delete plugin addition, then update the version from 3.7.1 to the latest.

Reply:
Changed maven-compiler-plugin 3.7.1 to 3.8.1

Reply:
Nice Work.
We still have the duplicate plugin.  Kindly remove one node

```
<plugin>
          <artifactId>maven-compiler-plugin</artifactId>
          <version>3.8.0</version>
          <version>3.8.1</version>
        </plugin>
```
```
<plugin>
<artifactId>maven-site-plugin</artifactId>
          <version>3.8.1</version>
        </plugin>
```
check here to see the latest version:  https://mvnrepository.com/artifact/org.apache.maven.plugins/maven-site-plugin

Reply:
Second comment mentioned maven-compiler-plugin. I checked https://mvnrepository.com/artifact/org.apache.maven.plugins/maven-compiler-plugin and 3.8.1 is the latest version. 

Last comment included the URL link for maven-compiler-plugin: https://mvnrepository.com/artifact/org.apache.maven.plugins/maven-site-plugin and 3.9.1 is the latest version. 

Can you confirm what plugin needs to be updated and what needs to be removed completely?

Reply:
Awesome stuff @tadeoye 
Good spot. There was no duplicate. It was my mistake.

No plugin needs to be removed. Update the `maven-site-plugin` to the latest. 👍

Reply:
Have updated `maven-site-plugin` in two places. Please review.